recipes-core/initrdscripts/initramfs-stubdomain/init.sh

```diff
@@ -82,7 +82,12 @@ dmargs=$( xenstore-read $(xenstore-list -p "$vm_uuid/image/dm-argv" | sort ))
 echo "target $target vm_uuid $vm_uuid"
 echo "Invoking qemu with dmargs       = ${dmargs}"
 
+# $dm_args and $kernel are separated with \n to allow for spaces in arguments.
+IFS=$'\n'
```

Comment:
```
OIFS=$IFS
IFS='\n'

... do whatever ...
IFS=$OIFS
```
```unset``` will clear IFS in the current scope, but IFS comes in with an initial value.  In the past I've saved the incoming IFS and then restored it when I was done handling the data to avoid clobbering it.

Reply:
Sounds good.  I'll make the change.